<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Product</class>
 <widget class="QDialog" name="Product">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>511</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="name">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="description">
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="price">
      <property name="text">
       <string>Price</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="stock_available">
      <property name="text">
       <string>Stock Available</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="name_input"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="price_input"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="stock_available_input"/>
    </item>
    <item row="1" column="1">
     <widget class="QTextEdit" name="description_input"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="add_product">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>360</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Product</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
